TMS_Tests/Core/Checkbox.cs

```diff
@@ -0,0 +1,75 @@
+﻿using OpenQA.Selenium;
+using TMS_Tests.Element;
+using TMS_Tests.Utils;
+
+namespace TMS_Tests.Core
+{
+    public class Checkbox
```

Comment:
Checkbox and RadioButton classes should be in Element folder

Reply:
Updated